lib/aytests/image_builder.rb

```diff
@@ -74,7 +74,11 @@ def cleanup_environment
               # Either the file does not exists anymore or there are cloned instances
               # which have not been removed correctly by previous run.
               log.info "CLEANUP: Removing unneeded file #{pathname} in pool #{pool}"
-              Cheetah.run(["sudo", "virsh", "vol-delete", pathname])
+              begin
+                Cheetah.run(["sudo", "virsh", "vol-delete", pathname])
+              rescue
```

Comment:
Cheetah can help you, if you `rescue Cheetah::ExecutionFailed`: https://github.com/openSUSE/cheetah/blob/2f5ace24fe9081a9033df8a00f4244bf746cd391/spec/cheetah_spec.rb#L640